### DISCUSSION TITLE
Problem consultation

### DISCUSSION THREAD

**Discussion by 2939672975** [bitcraze/discussions#891] [Q&A]
Dynamic capture releases the position and attitude information in real time through the vrpn protocol of ros. May I ask whether your company's UAV products provide the corresponding ros interface to receive and control the position and attitude information sent by the dynamic capture?

**Comment by krichardsson** [comment]
Currently we do not have direct support for ROS in our platform, but you can use [Crazyswarm2](https://github.com/IMRCLab/crazyswarm2) for this purpose and use external position information, for instance from a mocap system

**Reply by 2939672975** [reply]
Sorry to bother you, but we have a few more questions we'd like to ask you. First, can it be combined with the dynamic capture system that publishes the position and attitude information of each rigid body through the vrpn protocol under ros? Second, is our platform a ROs-based system with no direct ros communication interface, but is the system ROs-based? Third, do you have the video of the combination of drones and capture?

**Comment by krichardsson** [comment]
> First, can it be combined with the dynamic capture system that publishes the position and attitude information of each rigid body through the vrpn protocol under ros? 

Unfortunately I don''t know that much about Crazyswarm, please ask this in their [discussion forum](https://github.com/IMRCLab/crazyswarm2/discussions) 

Maybe you can find something in their documentation here https://imrclab.github.io/crazyswarm2/

> Second, is our platform a ROs-based system with no direct ros communication interface, but is the system ROs-based?

The Crazyflie is not ROS based, Crazyswarm2 provides a library that connects to the Crazyflie and exposes a ROS API to communicate with it.

> Third, do you have the video of the combination of drones and capture?

You can find lots of examples on our [youtube channel](https://www.youtube.com/@BitcrazeVideos), some of them uses mocap + Crazyswarm. Another examples is https://www.youtube.com/watch?v=ezTayb76x9U&ab_channel=ACTLabatBrown
